ldk/javascript/examples/self-test-loop/src/tests/system/index.ts

```diff
@@ -14,3 +14,24 @@ export const testOperatingSystem = (): Promise<boolean> =>
         reject(error);
       });
   });
+
+export const testGetEnvironment = (): Promise<boolean> =>
+  new Promise((resolve, reject) => {
+    system
+      .getEnvironment()
+      .then((getEnvironment) => {
+        console.debug(`System Aptitude: getEnvironment: osVersion: ${getEnvironment.osVersion}`);
+        console.debug(
+          `System Aptitude: getEnvironment: oliveHelpsVersion: ${getEnvironment.oliveHelpsVersion}`,
+        );
+        console.debug(
+          `System Aptitude: getEnvironment: loopVersion: ${getEnvironment.loopVersion}`,
+        );
+        setTimeout(() => {
```

Comment:
instead of just resolving true after a second could we add a check for each property and if each check passes then we resolve true.

I think the setTimeout should reject if the test doesnt pass within a certain amount of time

Reply:
Instead of display the message in console, can we create a whisper and show `osVersion`, `oliveHelpsVersion` and `loopVersion` there and provide buttons for `pass` or `fail`, something like this:
```
     <oh-box
            direction={whisper.Direction.Horizontal}
            justifyContent={whisper.JustifyContent.SpaceEvenly}
          >
            <oh-button label="Pass" onClick={() => resolve(true)} />
            <oh-button label="Fail" onClick={() => reject()} />
          </oh-box>